Fastpack/Cache.rei

```diff
@@ -0,0 +1,27 @@
+type t
+type load =
+  | Empty
+  | Load(params)
+and params = {
+  currentDir: string,
+  projectRootDir: string,
+  mock: list((string, Config.Mock.t)),
+  nodeModulesPaths: list(string),
+  resolveExtension: list(string),
+  preprocess: list(Preprocessor.config),
+};
+
+let make: load => Lwt.t(t);
+let isLoadedEmpty: t => bool;
+let addModule: (Module.t, t) => unit;
+let getModule: (Module.location, t) => Lwt.t(option(Module.t));
+let removeModule: (Module.location, t) => unit;
+let save: t => Lwt.t(unit);
+
+module File: {
```

Comment:
why grouping those functions under a module? I think it hurts discoverability (in autocomplete for example), there's nothing special about those function, they can be at the top level with names with `*File` suffix — `statFile` / `readFile` and so on.

Reply:
Well, it doesn't hurt autocomplete (at least in vim), but also I like it aesthetically.

Reply:
I mean you don't see those immediately when you do `Cache.<TAB HERE>`.

> also I like it aesthetically

well...

Reply:
Well, "aesthetics" argument is mostly for fun, but I do see it in scenario you have described